docs/data/date-pickers/calendar-systems/AdapterMomentJalali.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import moment from 'moment-jalaali';
+import 'moment/locale/fa';
+import { AdapterMomentJalaali } from '@mui/x-date-pickers/AdapterMomentJalaali';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import useTheme from '@mui/system/useTheme';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+export default function AdapterMomentJalali() {
+  moment.loadPersian({ dialect: 'persian-modern' });
+
+  const existingTheme = useTheme();
+  const theme = React.useMemo(
+    () => createTheme({ direction: 'rtl' }, existingTheme),
+    [existingTheme],
+  );
+
+  return (
+    <ThemeProvider theme={theme}>
+      <div dir="rtl">
+        <LocalizationProvider dateAdapter={AdapterMomentJalaali} adapterLocale="fa">
+          <DateTimePicker
+            label="AdapterMomentJalaali"
+            defaultValue={moment('2022-02-01T12:00:00')}
```

Comment:
`date-fns-jalaali` defaults to 12hr clock, whereas `moment-jalaali` to 24hr clock.
I set both to mid-day to at least have them show `12:00`.
Do you have better suggestions?
Maybe force `ampm` to `false` or `true` on both? 🤔

Reply:
Not sure if adding an exception to our AMPM management to fix this inconsistency is worth it
I think your test is nice and we can keep the behavior as it is